src/tcpconns.c

```diff
@@ -171,6 +176,24 @@ struct tcpi_field_selector {
 static struct tcpi_field_selector *tcpi_fields_to_report = NULL;
 static size_t num_tcpi_fields_to_report = 0;
 static size_t tcpi_fields_to_report_size = 0;
+static size_t num_tcpi_counter_fields = 0;
+
+static uint64_t counter_max_delta = 1000000;
+
+/* Cache for computing deltas of DELTA values. */
+static cdtime_t counter_cache_timeout = DOUBLE_TO_CDTIME_T(120.0);
```

Comment:
Consider replacing by some sort of mechanism that counts conn_read calls and delete entries that didn't appear in the last completed conn_read call